client/src/api/spotify.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+export const spotifyConnect = (spotifyId, spotifySecret) => {
+  return axios('https://accounts.spotify.com/api/token', {
```

Comment:
The client is still making this request. All backend logic - talking to spotify, etc. should happen from our backend server. So this should be moved to the server. Here we should make a request to our server which will then make the request to spotify and return the desired data.